### ISSUE TITLE
Update bin-wrapper to 4.0

### ISSUE THREAD

**Issue by ralphsaunders** [imagemin/pngquant-bin#90]
Hi there,

Is it possible to see pngquant-bin update their [bin-wrapper](https://www.npmjs.com/package/bin-wrapper) dependency?

Currently the dependency tree for `imagemin-pngquant` and `pngquant-bin` still relies on `gulp-util`:

```
├─┬ imagemin-pngquant@6.0.0
│ └─┬ pngquant-bin@5.0.0
│   └─┬ bin-wrapper@3.0.2
│     └─┬ download@4.4.3
│       └─┬ gulp-decompress@1.2.0
│         └── gulp-util@3.0.8  deduped
```

If `bin-wrapper` is updated to 4.0 this will cease to be an issue.

It looks like the build is failing for #89 which addresses this issue.

Thank you,
Ralph

**Comment by patsplat** [comment]
There's also a `node audit` vulnerability reported for bin-wrapper 3.x

**Comment by Diaoul** [comment]
This will fix #82 also as `download` is stuck to 6.x (issue fixed in 7.1) for bin-wrapper 3.x.

**Comment by LOK-Soft** [comment]
added PR #95 which is a corrected version of the already mentioned one
